atlasdb-client/src/main/java/com/palantir/atlasdb/keyvalue/impl/DualWriteKeyValueService.java

```diff
@@ -110,6 +110,12 @@ public void delete(TableReference tableRef, Multimap<Cell, Long> keys) {
     }
 
     @Override
+    public void deleteRange(TableReference tableRef, RangeRequest range) {
```

Comment:
If first throws, transaction should not commit, so one would need to retry delete